src/main/java/org/example/OccurrenceWordPoc.java

```diff
@@ -19,11 +27,15 @@ public static void main(String[] args) {
                 if (inputFile.exists()) {
                     System.out.println("File already exists");
                 } else {
+                    DatabaseHelper databasehelper = new DatabaseHelper();
+                    databasehelper.storingDataToDataBase(inputFilePath, searchWord, "Error", 0, "File not found");
                     System.out.println("File dont exist");
                     return;
                 }
                 performReadWordSearch(inputFilePath, searchWord);
```

Comment:
Shouldn't this method "performReadWordSearch" be called only if file exists? If the answer is yes, then this method should be called before line no 29.